astropy/cosmology/_io/mapping.py

```diff
@@ -17,12 +17,12 @@
 __all__ = []  # nothing is publicly scoped
 
 
-def from_mapping(map, *, move_to_meta=False, cosmology=None):
+def from_mapping(mapping, /, *, move_to_meta=False, cosmology=None):
```

Comment:
What does this have to do with renaming `astropy.cosmology.io`?

Reply:
General improvement. These are private functions and can be changed. In this case it's changing from overriding a builtin to a better name.

Reply:
This change to the code is an improvement, I just didn't expect it because the pull request description only talks about renaming `astropy.cosmology.io`. Could you edit the pull request so that it would describe all the changes?

Reply:
Done!